### PR TITLE
fix misspelling across codebase

### PR DESCRIPTION
**Pull request by umangachapagain**
Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

File: controllers/util/predicates.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 )
 
-// ComposePredicates will compose a variable number of predicte and return a predicate that
+// ComposePredicates will compose a variable number of predicate and return a predicate that
```
Comment (obnoxxx) [inline controllers/util/predicates.go:10]:
```suggestion
// ComposePredicates will compose a variable number of predicates and return a predicate that
```

**Review by obnoxxx — CHANGES_REQUESTED** [review]
Good catches, but I think there's still one error. See inline.

**Comment by umangachapagain** [comment]
> Good catches, but I think there's still one error. See inline.

Gotta catch 'em all.

**Review by obnoxxx — APPROVED** [review]
/lgtm

**Pull request merged**